nebu/cli/mathify.py

```diff
@@ -0,0 +1,99 @@
+import os
+from pathlib import Path
+import re
+
+import click
+import docker
+
+from ..logger import logger
+from ._common import common_params
+
+
+IMAGE_TAG = 'openstax/cnx-bakedpdf'
+REPO_URL = 'https://github.com/openstax/baked-pdf.git'
```

Comment:
You renamed this repo yesterday. I'm sure the link still works, so it'd be an easy thing to miss when going over this code again.